client/src/components/common/Button/index.js

```diff
@@ -0,0 +1,4 @@
+import ContainedButton from './ContainedButton';
+import OutlinedButton from './OutlinedButton'
+
+export  {ContainedButton, OutlinedButton};
```

Comment:
there is no need for these line here, just create Button component here in this file , 
so you have to push only 2 files to this pull request, one is `index.js` for Button component & the other for `style.js` and add the style that you will imported to `index.js` as well.